CHANGELOG.md

```diff
@@ -5,6 +5,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to the versioning scheme outlined in the [README.md](README.md).
 
+## [Unreleased]
+
+### Changed
+
+- The pessimistic execution cost estimator differentiates between
+  contracts with different origin addresses.
```

Comment:
Needs to be fleshed out with the fee estimation work and mempool sync

Reply:
Yup made a separate PR for changelog: https://github.com/blockstack/stacks-blockchain/pull/2997